tripal_chado/src/Plugin/Field/FieldWidget/ChadoPropertySelectWidgetDefault.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Drupal\tripal_chado\Plugin\Field\FieldWidget;
+
+use Drupal\tripal\TripalField\TripalWidgetBase;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\tripal_chado\TripalField\ChadoWidgetBase;
+
+/**
+ * Plugin implementation of default Tripal linker property widget.
+ *
+ * @FieldWidget(
+ *   id = "chado_property_select_widget_default",
+ *   label = @Translation("Chado Property: Select Drop-down"),
+ *   description = @Translation("Provides a configurable drop-down widget for Chado Properties."),
+ *   field_types = {
+ *     "chado_property_type_default"
+ *   }
+ * )
+ */
+class ChadoPropertySelectWidgetDefault extends ChadoWidgetBase {
+
+
+  /**
+   * {@inheritdoc}
+   */
+  public function formElement(FieldItemListInterface $items, $delta, array $element, array &$form, FormStateInterface $form_state) {
+
+    // Get the field settings.
+    $field_definition = $items[$delta]->getFieldDefinition();
+    $field_settings = $field_definition->getSettings();
+
+    // Get the default values.
+    $item_vals = $items[$delta]->getValue();
+    $record_id = $item_vals['record_id'] ?? 0;
+    $prop_id = $item_vals['prop_id'] ?? 0;
+    $linker_id = $item_vals['linker_id'] ?? 0;
+    $default_value = $item_vals['value'] ?? NULL;
+    $term_id = NULL;
+    if ($field_settings['termIdSpace'] and $field_settings['termAccession']) {
+      $idSpace_manager = \Drupal::service('tripal.collection_plugin_manager.idspace');
+      $idSpace = $idSpace_manager->loadCollection($field_settings['termIdSpace']);
+
+      $term = $idSpace->getTerm($field_settings['termAccession']);
+      $term_id = $term->getInternalId();
+    }
+
+    // Process the options from the configuration.
+    $raw_options = $this->getSetting('options');
+    $options = [];
+    foreach( explode("\n", $raw_options) as $item) {
+      $item = trim($item);
+      $options[$item] = $item;
+    }
+
+    // Check that the default value is one of the available options.
+    if (isset($default_value) and !array_key_exists($default_value, $options)) {
+      $request_method = \Drupal::requestStack()->getCurrentRequest()->getMethod();
+      // Do not display the warning when saving
+      if ($request_method == 'GET') {
```

Comment:
```suggestion
      // Do not display the warning when saving
      if (!$form_state->getUserInput()) {
```
Cleaner version without the extra service